tests/test_http.py

```diff
@@ -0,0 +1,164 @@
+import logging
+
+from flexmock import flexmock
+import pycurl
+import pytest
+
+import osbs.http as osbs_http
+from osbs.http import parse_headers, HttpSession, HttpStream
+
+from tests.fake_api import Connection, ResponseMapping
+
+logger = logging.getLogger("osbs.tests")
```

Comment:
I don't know, I copy-pasted that from the previous version of tests :) I will fix that.